### ISSUE TITLE
“Bias squared” rather than “Bias”?

### ISSUE THREAD

**Issue by huanfachen** [rasbt/mlxtend#1083]
I have a question about the “Average bias” in [this documentation](https://rasbt.github.io/mlxtend/user_guide/evaluate/bias_variance_decomp/) and [the coding of the bias_variance_decomp function](https://github.com/rasbt/mlxtend/blob/master/mlxtend/evaluate/bias_variance_decomp.py).

If Loss = [Bias]^2 + Variance, then in the reporting of the bias_variance_decomp function, it should be **average bias squared** rather than **average bias**?

Any comment is welcome, many thanks!

**Comment by rasbt** [comment]
Yes, that's a good point. It's average biased squared. Maybe it should be made more explicit in the docs like you suggest.
